concepticondata/conceptlists/Miller-1984-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Miller-1984-100-1	1	no	not, no	1240	NOT
+Miller-1984-100-2	2	muchos	many	1198	MANY
+Miller-1984-100-3	3	uno	one	1493	ONE
+Miller-1984-100-4	4	dos	two	1498	TWO
+Miller-1984-100-5	5	grande	big	1202	BIG
+Miller-1984-100-6	6	viejo	old	1229	OLD
+Miller-1984-100-7	7	chico	small	1246	SMALL
+Miller-1984-100-8	8	largo	long	1203	LONG
+Miller-1984-100-9	9	nuevo	new	1231	NEW
+Miller-1984-100-10	10	bueno	good	1035	GOOD
+Miller-1984-100-11	11	pesado	heavy	1210	HEAVY
+Miller-1984-100-12	12	seco	dry	1398	DRY
+Miller-1984-100-13	13	frío	cold	2483	COLD (OF WEATHER)
```

Comment:
Frío is usually mapped to COLD, is there a special reason you went for COLD (OF WEATHER) here?

Reply:
I think I should change it to COLD as this list is claimed to bear a strong resemblance to Swadesh 1955.